apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,84 @@
+---
+title: Automatically Stop and Start App V2 Fly Machines
+objective: 
+layout: docs
+nav: firecracker
+order: 65
+---
+
+<div class="callout">
+This feature only works for V2 apps. For information about scaling V1 apps, refer to [Scale V1 Nomad Apps](/docs/apps/legacy-scaling/).
+</div>
+
+You can decide whether you want to automatically stop an app's Fly Machines when they're idle and automatically restart them when they're needed. Fly Proxy controls automatically starting and automatically stopping Machines according to settings in your `fly.toml` file, in either the `services` or `http_service` sections:
```

Comment:
Suggested fodder:

Fly Machines are fast to start and stop, and you don't pay for their CPU and RAM when they're in the `stopped` state. For Fly Apps with a [service](/docs/reference/configuration/#the-services-sections) configured, Fly Proxy can start and stop existing Machines based on incoming requests, so that your app can accommodate bursts in demand without keeping extra Machines running constantly.

You can configure automatic starts and automatic stops independently with the `auto_start_machines` and `auto_stop_machines` settings within the `[[services]]` or `[http_service]` sections of `fly.toml`, and tune them with the service's [concurrency limits](/docs/reference/configuration/#services-concurrency).

Reply:
This is great. Tweaked a bit to get links lined up.